pkgs/top-level/ocaml-packages.nix

```diff
@@ -882,14 +882,15 @@ let
             }
           else
             import ../development/ocaml-modules/janestreet {
-            };
+            }
+        );
 
-        janeStreet_0_15 =
+        janeStreet_0_15 = lib.recurseIntoAttrs (
```

Comment:
What’s the purpose of this change? The purpose of this sub-package-set is to support selected packages such as `bap` or `biocaml`. Requiring that it fully evaluates and build seems counter-productive to me.

Reply:
The intention is that it gets checked by ci and no longer has an eval breaking change.